media/stylus/includes/vars.styl

```diff
@@ -173,7 +173,7 @@ $selector-heading-font-fallback = 'html[data-ffo-opensanslight="false"]:not(.no-
 paths
 ====================================================================== */
 $path-to-assets = '/media/';
-$path-to-images = $path-to-assets + 'img/';
+$path-to-images = '../../static/img/';
 $logo-sprite-url = $path-to-images + 'logo_sprite.svg?2014-01';
```

Comment:
I think this should be:

```
$path-to-assets = '../';
$path-to-images = $path-to-assets + 'img/';
```

I hope I'm not missing something, let's look at an example. There is this in compat-icons.styl:

```
url($path-to-assets + 'fonts/compat-icons.eot?-ehmwve')
```

The Stylus file will be compiled into `<STATIC_ROOT>/css/wiki-compat-tables.css` so eventually be accessible via `/static/css/wiki-compat-tables.css`. With the current code it'll have this hardcoded:

```
url("/media/fonts/compat-icons.eot?-ehmwve")
```

But we want to make the Stylus files completely unaware how the compiled files are hosted (as that's an implementation detail and outside the scope of what Stylus should know) and so we need to make all paths in the compiled files relative.

So with `$path-to-assets = '../';` we'll get this:

```
url("../fonts/compat-icons.eot?-ehmwve")
```

Which will work since the browser will just traverse the URL path up from the location of the compiled file at `/static/css/wiki-compat-tables.css`